index.html

```diff
@@ -3,10 +3,15 @@
 <head>
   <meta charset="UTF-8">
   <title>Superhero Phone Book</title>
+  <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.6/angular.min.js"></script>
+ <script src="app.js"></script>
   <link rel="stylesheet" href="style.css">
 </head>
-<body>
-
-  <script src="app.js"></script>
+<body ng-controller='heroesCtrl as vm'>
+<input type='text' ng-model='search' placeholder="Search">
+<button ng-click="vm.addHeroes()">Search</button>
+<div ng-repeat="hereos in vm.heroes">
+<p>{{$index + 1}}. {{heroes.name}}</p>
+</div>
 </body>
 </html>
```

Comment:
When you have time I would recommend going back to this lab and implementing steps 7 onward to get in the practice working within a simple single page application using ng.